### PR TITLE
IMBU-5: Add web service configuration

### PR DESCRIPTION
**Pull request by lscheinkman**
@jcasner This is the web version. 
I have a small document with the extra steps needed to bring a demo server up using our internal salt roles.
FYI: @BoltzmannBrain, @unixorn

### CODE REVIEW

File: imbu/conf/supervisord.conf [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+[rpcinterface:supervisor]
+supervisor.rpcinterface_factory = supervisor.rpcinterface:make_main_rpcinterface
+
+[supervisorctl]
+serverurl=http://:
+
+;*************** FLUENT-API **************
+[program:fluent-api]
+command=uwsgi --enable-threads --socket 0.0.0.0:19002 --master --vacuum --idle 300 --processes 8 --threads 4 --listen 1024 --wsgi-file engine/fluent_api.py
```
Comment (oxtopus) [inline imbu/conf/supervisord.conf]: Given the scope of imbu, I think we can safely reduce `--listen` to a much lower level that doesn't require system-level configuration to support it.

Comment (lscheinkman) [inline imbu/conf/supervisord.conf]: Good point. 
Should I just remove this extra parameter and rely on `uwsgi` defaults? I think the default value is 100.

Comment (oxtopus) [inline imbu/conf/supervisord.conf]: +1 for default value.  Can also reduce `--processes` to 2 or 1.

**Comment by jcasner** [comment]
:+1:

**Pull request merged**